.travis.yml

```diff
@@ -17,13 +22,19 @@ cache:
 before_script:
   - curl -sSL https://github.com/maidsafe/QA/raw/master/travis/cargo_install.sh > cargo_install.sh
   - bash cargo_install.sh cargo-prune;
-  - rustup component add rustfmt
-  - rustup component add clippy
+  - rustup target add $TARGET
+  - rustup component add rustfmt clippy
+  - if [ "${TRAVIS_OS_NAME}" = osx ]; then
+      mkdir -p .cargo;
+      echo '[target.x86_64-unknown-linux-musl]
```

Comment:
Why not create the `.cargo/config` file locally?

Reply:
Even with `musl-tools` installed? Marking as resolved as it was just a suggestion.